eclass/qt5-build.eclass

```diff
@@ -179,8 +179,14 @@ qt5-build_src_prepare() {
 		qt5_symlink_tools_to_build_dir
 
 		# Avoid unnecessary qmake recompilations
-		sed -i -re "s|^if true;.*(\[ '\!').*(\"\\\$outpath/bin/qmake\".*)|if \1 -e \2 then|" \
-			configure || die "sed failed (skip qmake bootstrap)"
+		if [[ ${QT5_MINOR_VERSION} -ge 8 ]]; then
+			sed -i -e '/Creating qmake/i if false; then' \
```

Comment:
hmm this disables the build unconditionally though right? how does qmake get built in `qtcore`?